src/core/server/ui_settings/create_or_upgrade_saved_config/create_or_upgrade_saved_config.test.ts

```diff
@@ -136,20 +135,21 @@ describe('uiSettings/createOrUpgradeSavedConfig', function() {
       });
 
       await run();
-      sinon.assert.calledOnce(logWithMetadata);
-      sinon.assert.calledWithExactly(
-        logWithMetadata,
-        ['plugin', 'elasticsearch'],
```

Comment:
How it's critical if we change message tags? Should we consider them as breaking changes? Not sure we even want to add such things to our release notes.

Reply:
I think it's OK during the 7.x timeline as we don't really respect breaking changes. I'd say we want to get these breaks in now so we can establish better breakage heuristics for 8.x.

Reply:
We have in the past considered logging changes to be breaking changes, but I agree with @rudolf that debug-level is probably not subject to that.

Reply:
ok, I thought that probably **Kibana devs** track some statistic & metrics for debug info and it **could** be important. Although it's hard to say now, so probably it's not worth bothering.